x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/action/LocalStateSQLXpackPlugin.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.sql.action;
+
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.xpack.core.LocalStateCompositeXPackPlugin;
+import org.elasticsearch.xpack.sql.plugin.SqlLicenseChecker;
+import org.elasticsearch.xpack.sql.plugin.SqlPlugin;
+
+import java.nio.file.Path;
+
+public class LocalStateSQLXpackPlugin extends LocalStateCompositeXPackPlugin {
```

Comment:
Does it have to be `public`? Also, I think the naming around xpack should be XPack (capital `P`).

Reply:
no, can be package private, thx!

Reply:
Actually, it needs to be public:
```
   > Throwable #1: java.lang.IllegalStateException: no public constructor for [org.elasticsearch.xpack.sql.action.LocalStateSQLXPackPlugin]
   >    at org.elasticsearch.plugins.PluginsService.loadPlugin(PluginsService.java:590)
   >    at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:119)
   >    at org.elasticsearch.node.Node.<init>(Node.java:303)
```